lib/internal/modules/esm/get_source.js

```diff
@@ -33,9 +37,20 @@ async function defaultGetSource(url, { format } = {}, defaultGetSource) {
       throw new ERR_INVALID_URL(url);
     }
     const { 1: base64, 2: body } = match;
-    source = Buffer.from(decodeURIComponent(body), base64 ? 'base64' : 'utf8');
+    source = BufferFrom(decodeURIComponent(body), base64 ? 'base64' : 'utf8');
+  } else if (experimentalNetworkImports && (
+    parsed.protocol === 'https:' ||
+    parsed.protocol === 'http:'
+  )) {
+    const res = await fetch(parsed, context);
+    const source = await res.body;
+    return source;
```

Comment:
@bmeck should this be early returning and skipping the policy manifest check?

If not, it would be this:

```suggestion
    source = await res.body;
```

Reply:
nice catch, it should not be returning early.